napps/kytos/of_flow_manager/README.rst

```diff
@@ -32,20 +32,21 @@ Protocol Operation
 
 The endpoints implemented in this napp are presented in the table below.
 
-+----------------------------------------+----------------------------------+------------+
-| Endpoint                               | Description                      | Method     |
-+========================================+==================================+============+
-| ``/flow-manager/dpid/flows``           | Retrieve flows from a specific   | ``GET``    |
-|                                        | Switch                           |            |
-+----------------------------------------+----------------------------------+------------+
-| ``/flow-manager/flows``                | Retrieve all flows               | ``GET``    |
-+----------------------------------------+----------------------------------+------------+
-| ``/flow-manager/dpid/flows-a``         | Add a flow in a specific Switch  | ``POST``   |
-|                                        |                                  |            |
-+----------------------------------------+----------------------------------+------------+
-| ``/flow-manager/dpid/flow_id/flows-d`` | Delete a flow from a specific    | ``DELETE`` |
-|                                        | Switch                           |            |
-+----------------------------------------+----------------------------------+------------+
++------------------------------------------+----------------------------------+------------+
+| Endpoint                                 | Description                      | Method     |
++==========================================+==================================+============+
+| ``/flow-manager/flows[/dpid]``           | Retrieve installed flows from    | ``GET``    |
```

Comment:
Yes, you can query every switch without specifying the dpid, so I think the syntax is correct.

About the POST / DELETE rules, it's dangerous for us somehow? Whoever comes to access the endpoints can see in the readme or examples file how to do it correctly, and implement something to send failsafe requests, imho.